blocklylib-core/src/main/java/com/google/blockly/android/WorkspaceFragment.java

```diff
@@ -52,16 +52,27 @@
 public class WorkspaceFragment extends Fragment {
     private static final String TAG = "WorkspaceFragment";
 
-    public static final boolean DEFAULT_SCROLLABLE = true;
+    //no scrollbar, no zoom, no buttons
```

Comment:
Since the AbstractBlocklyActivity will need access to these values also, move these constants to a new public final class ZoomBehavior.